pythonforandroid/bootstraps/sdl2/build/build.py

```diff
@@ -488,6 +488,9 @@ def parse_args(args=None):
 
     if args.permissions is None:
         args.permissions = []
+    elif args.permissions:
+        if isinstance(args.permissions[0], list):
+            args.permissions = [p for perm in args.permissions for p in perm]
```

Comment:
I think I don't understand. It's a list comprehension, it _wants_ the returning variable like that, otherwise it'd be this:

    _new_permissions = []
    for permission in args.permissions:
        for perm in permission:
            _new_permissions.append(perm)
    args.permissions = _new_permissions

and repeated three times (pygame+sdl2+webview).